app/admin/event.rb

```diff
@@ -0,0 +1,71 @@
+ActiveAdmin.register Event do
+  permit_params :subject, :datetime, :confirmed, :location_id,
+                :user_events, :users
+
+  index do
+    selectable_column
+    column :id
+    column :subject
+    column :datetime
+    column :confirmed
+    column :location
+    column :created_at
+
+    ##########
+    column :speakers do |event|
+      table_for event.users.each do
+        column do |user|
+          link_to user.email, [:admin, user]
+        end
+      end
+    end
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :subject
+      row :location
+      table_for event.users.each do
+        column do |user|
+          link_to user.email, [:admin, user]
+        end
+      end
+    end
+  end
+
+  form do |f|
+    f.inputs 'Event' do
+      f.semantic_errors(*f.object.errors.keys)
+      f.input :subject
+      f.input :datetime, as: :datetime_picker
+      f.input :location
+      f.input :confirmed
+
+      ##########
+      # f.input :users, as: :select, multiple: true,
+      # collection: User.all.map {|q| [q.email, q]}
+
+      # f.input :users,
+      # collection: User.all.map{ |user| [user.email, user.id] },
+      # multiple: 'multiple'
+      # f.input :users, as: :check_boxes
+    end
+    f.actions
+  end
+
+  filter :confirmed, as: :check_boxes
+
+  scope :all, default: true
+  scope :today do |events|
+    events.where('? < datetime and datetime < ?',
+                 DateTime.now.to_date - 1.days, DateTime.now.to_date + 1.days)
```

Comment:
few comments. also read comment below.
1. for such queries you can use Range:
events.where(datetime: one_date..second_date)
2. better to use other scope for querying today events:
DateTime.now.beginning_of_day, DateTime.now.end_of_day